pkg/endpoint/bpf.go

```diff
@@ -59,6 +62,50 @@ const (
 	EndpointGenerationTimeout = 55 * time.Second
 )
 
+// mapPath returns the path to a map for endpoint ID.
+func mapPath(mapname string, id int) string {
```

Comment:
Can you make a commit that makes this function a method of `Endpoint`?

```golang
func (e *Endpoint) mapPath(mapName string) string {
	return bpf.MapPath(mapname + strconv.Itoa(e.ID))
}
```

Reply:
Done.